diffraction.html

```diff
@@ -205,9 +205,8 @@ <h4><b>Experimental procedure</b></h4>
 
                             <div id="menu2" class="tab-pane fade">
                               <h3>VIDEO</h3>
-                              <p align="center" style="height: 200px">CURRENTLY NOT AVAILABLE
-                              <br>
-                              <img src="resources/images/sad_emoji.png" style="height: 100px"></p>
+                            <p align="center"><iframe width="500.1vw" height="280vh" src="https://www.youtube.com/embed/uohd0TtqOaw" title="Diffraction Demo: Single Slit and Circular Aperture" frameborder="0" allow="autoplay; encrypted-media;" allowfullscreen></iframe></p>
```

Comment:
@dhruv302003 any reason for choosing `width="500.1vw"`? 
Let's keep it 500vw otherwise to keep things consistent. Make this change in other files as well.